frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -357,6 +363,35 @@ export function FeatureFlag({ id }: { id?: string } = {}): JSX.Element {
     )
 }
 
+function ExposureTab({ featureFlagKey }: { featureFlagKey: string }): JSX.Element {
+    return (
+        <EventsTable
+            fixedFilters={{
+                event_filter: '$feature_flag_called',
+                properties: [
+                    {
+                        key: '$feature/' + featureFlagKey,
+                        type: 'event',
+                        value: ['true'],
```

Comment:
Looks like this won't work for multivariate flags?

Reply:
Thinking the same for recordings, we might want to have a variant selector?

Reply:
ah, good point

Reply:
Maybe we catch all the cases by doing is_not false?

Reply:
good idea! and also not undefined / not null, although not sure if we ever send that, would have to check

Reply:
not for this PR, but I think for the recordings panel it's worth choosing the variant you want to see?

Reply:
yeah, agreed. I think there's a lot to improve off both these features. What i'm testing for is usage/interaction. From the handful of interactions i'm seeing, recordings haven't even been enabled.

Reply:
From doing some filtering it doesn't look like anything ends up null/undefined. Will refrain from adding those filters for now